pkg/utils/consts/consts.go

```diff
@@ -49,6 +50,22 @@ const (
 	GpuSkuPrefix = "Standard_N"
 
 	NodePluginInstallTimeout = 60 * time.Second
+
+	// PortInferenceServer is the default port for the inference server.
+	PortInferenceServer = int32(5000)
```

Comment:
I think we must have defined this port somewhere in the code base, can you double check?